test/python/algorithms/test_grover.py

```diff
@@ -95,41 +97,63 @@ def setUp(self):
         self.qasm = QuantumInstance(
             BasicAer.get_backend("qasm_simulator"), seed_simulator=12, seed_transpiler=32
         )
+        self._sampler = Sampler()
```

Comment:
By default Sampler() will return an ideal distribution right, i.e. is more akin to the former statevector result. Should we be also testing using say shots=1024 which would have been the default before for qasm simulation?